stellarphot/differential_photometry/aij_rel_fluxes.py

```diff
@@ -47,7 +48,7 @@ def calc_aij_relative_flux(star_data, comp_stars,
 
```

Comment:
Missing documentation of following parameters in function definition 

- `in_place=True`
- `coord_column=None`

Reply:
I'm confused -- do you mean the default value isn't listed?  I'd prefer to just leave that in the function signature so that it doesn't have to be updated in two places if it changes in the future.

Reply:
I didn't see the parameters listed at all in the docs, or am I mistaken?

Reply:
They are tehre but were maybe originally not shown because they didn't change?

Reply:
I really hate this reviewing interface sometimes. 

I do think the default values should be listed because otherwise a person may think the interface behave differently without the parameter being passed.  Also, I believe I listed defaults for all the docstrings I created/added, which frankly was a lot of them in stellarphot. :)